Libplanet.Action/State/ReservedAddresses.cs

```diff
@@ -0,0 +1,16 @@
+using Libplanet.Crypto;
+
+namespace Libplanet.Action.State
+{
+    public static class ReservedAddresses
+    {
+        public static readonly Address LegacyAccount
+            = new Address("1000000000000000000000000000000000000000");
+
+        public static readonly Address FungibleAssetsAccount
+            = new Address("1000000000000000000000000000000000000001");
+
+        public static readonly Address ValidatorSetAddress
+            = new Address("1000000000000000000000000000000000000002");
```

Comment:
I'm not so sure about these. Seems like unused at the moment. We might want to have a more concrete design first. 🙄

Reply:
These will be handled on https://github.com/planetarium/libplanet/pull/3483.